hatice-ecevit-2016400138/assignment3/Dict.java

```diff
@@ -0,0 +1,84 @@
+import java.util.HashMap;
+
+public class Dict {

```

Comment:
If you didn't know, this is known as a Trie structure. It is good practice to add javadoc explaining this, and maybe the bigO performance of each method.